lib/engine/game/g_1880/step/buy_train.rb

```diff
@@ -41,7 +41,10 @@ def round_state
 
           def pass!
             train_name = @game.depot.upcoming.first.name
-            return super if train_name == '10' || train_name == '2P' || !discard_trains?
+            train_index = @game.depot.upcoming.first.index
+            if (train_name == '8E' && train_index == 1) || train_name == '10' || train_name == '2P' || !discard_trains?
```

Comment:
Can you get this back to a one-liner? Something like `['10', '2P'].include?(train_name)` should do it.